src/_pytest/junitxml.py

```diff
@@ -361,12 +371,15 @@ def mangle_test_address(address):
 
 
 class LogXML(object):
-    def __init__(self, logfile, prefix, suite_name="pytest", logging="no"):
+    def __init__(
+        self, logfile, prefix, suite_name="pytest", logging="no", report_duration=None
```

Comment:
To make things simpler I propose to use `report_duration="total"`; this way you can remove the `not self.report_duration` from the check in `update_testcase_duration`

Reply:
That was the initial version, but then I thought that duplicating const "total" at https://github.com/aparamon/pytest/blob/cfbd387a5d7a27a3e7dfb754d7cd4482e931822b/testing/test_junitxml.py#L748
would be less desirable. (Also overall, using `None` as default value looks good interface.)